crates/nu-cli/tests/commands/autoenv.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Noticed a few empty files. Were you thinking of adding some tests for trust/untrust as part of this?